test/hotspot/jtreg/compiler/lib/ir_framework/TestFramework.java

```diff
@@ -409,6 +410,17 @@ public TestFramework setDefaultWarmup(int defaultWarmup) {
         return this;
     }
 
+    /**
+     * In rare cases, methods may not be compilable because of a compilation bailout. By default, this leads to a
+     * test failure. However, if such cases are expected in a test class, this flag can be set to true, which
+     * allows the all test to pass even if there is no compilation. Any associated {@link IR} rule is only executed
+     * if the test method was compiled, and else it is ignored silently.
```

Comment:
/**
     * In rare cases, methods may not be compilable because of a compilation bailout. By default, this leads to a
     * test failure. However, if such cases are expected in multiple methods in a test class, this flag can be set to
     * true, which allows any test to pass even if there is a compilation bailout. If only selected methods are prone
     * to bail out, it is preferred to use {@link Test#allowNotCompilable()} instead for more fine-grained control.
     * By setting this flag, any associated {@link IR} rule of a test is only executed if the test method was compiled, 
     * and else it is ignored silently.

Reply:
Applied :)